src/wallet/users.spec.ts

```diff
@@ -251,6 +252,17 @@ describe("test user wallets", async () => {
         );
     });
 
+    it("should loadSecretKey, but without 'kind' field", async function () {
+        const testdataPath = path.resolve(__dirname, "..", "testdata/testwallets");
+        const keystorePath = path.resolve(testdataPath, "withoutKind.json");
+        const secretKey = UserWallet.loadSecretKey(keystorePath, password);
+
+        assert.equal(
+            secretKey.generatePublicKey().toAddress().bech32(),
```

Comment:
use `toBech32()`. can be done in the next PR.

Reply:
done